site/public/js/event-advertisements.js

```diff
@@ -0,0 +1,73 @@
+/* eslint no-undef: "off" */
+
+const eventLS = {
+    open: 'open',
+    bArr: 'bannerArray',
+    rArr: 'removedArray',
+    index: 'eventIndex',
+    duckTalk: 'duckTalking',
```

Comment:
I think it might make sense to do something like this because then the constants make more sense as to their purpose and I think just using 'open' as a key is probably not as unique as we would want.
```suggestion
    open: 'bannerOpen',
    bannerArr: 'bannerArray',
    removedArr: 'removedArray',
    index: 'eventIndex',
    duckTalk: 'duckTalking',
```